snuba/datasets/discover.py

```diff
@@ -275,6 +275,8 @@ def column_expr(
                 return "transaction_name"
             if column_name == "message":
                 return "transaction_name"
+            if column_name == "group_id":
```

Comment:
Ok, this could work in the short term. But in the long term we cannot rely on doing this for all fields that do not exist on a table. We will have to transform nonsensical conditions like this in to `False` or, better, return immediately an empty resultset. 

Could you please add a TODO that reminds us to come back here and turn nonsensical conditions into `false` instead of giving them to clickhouse anyway. ?